app/javascript/mastodon/components/hover_card_controller.tsx

```diff
@@ -23,19 +23,38 @@ export const HoverCardController: React.FC = () => {
   const [open, setOpen] = useState(false);
   const [accountId, setAccountId] = useState<string | undefined>();
   const [anchor, setAnchor] = useState<HTMLElement | null>(null);
-  const cardRef = useRef<HTMLDivElement>(null);
+  const cardRef = useRef<HTMLDivElement | null>(null);
+  const anchorRef = useRef<HTMLElement | null>(null);
+  const scrollingRef = useRef<boolean>(false);
```

Comment:
Maybe rename to `isScrollingRef`?

Also does it need to be defined as a ref? I wonder if this could be a `const` in `useEffect`, as it is only used there.